user-guide/notebooks/templates/template-accessing-the-data-directly.ipynb

```diff
@@ -149,7 +149,7 @@
    "source": [
```

Comment:
<p>So a user needs to fill in a collection here? presumably a string in quotes? I'm not so confident that's clear. Oh this is a template</p><p><code>collection_id = "put your collection here"</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>